tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -217,6 +217,10 @@ protected:
    /// are changed to their unencoded counterparts.
    void AutoAdjustColumnTypes(const RNTupleWriteOptions &options);
 
+   /// Factory method to resurrect a field from the stored on-disk type information.  This overload takes an already
+   /// normalized type name and type alias
+   static RResult<std::unique_ptr<RFieldBase>>
+   Create(const std::string &fieldName, const std::string &normalizedType, const std::string &typeAlias);
```

Comment:
Since both type names are normalize, how about
```suggestion
   Create(const std::string &fieldName, const std::string &canonicalType, const std::string &typeAlias);
```